### PR TITLE
Modifications to Documents

### PR DESCRIPTION
**Pull request by joshlay96**
- Changed Team name to "ET"
- Change product name to "EasyTravel"
- Changed team name and start date of project to Jan 2020 in DG

- Changed Parameter in UG for `editactivity` to `TITLE` instead of `Name`
- Update Command Summary

- And my own PPP as well lol

### CODE REVIEW

**Review by Deunitato — APPROVED** [review]

File: docs/UserGuide.adoc [hunk 1]
```diff
@@ -2240,6 +2243,9 @@ This feature will allow you to import and export your trip data in different for
 |*Check Progress* : `status`
 |`status`
 
+|*Vieww Itineary*: `listschedule`
+|`listschedule`
```
Comment (zenatrick) [inline docs/UserGuide.adoc:2247]: Remove this, can already be found in line 2285

File: docs/UserGuide.adoc [hunk 2]
```diff
@@ -2258,10 +2264,10 @@ This feature will allow you to import and export your trip data in different for
 |*Delete Activity*: `deleteactivity INDEX`
 |`deleteactivity 1`
 
-|*Sort Activities*: `sortactivity CRITERIA  ORDER`
+|*Sort Activities*: `sortactivity CRITERIA ORDER`
 |`sortactivity title ascending`
 
-|*Find Activities by Keywords*: `findactivity KEYWORD [MORE_KEYWORDS]...`
+|*Find Activities by Keywords*: `findactivity TITLE [MORE_TITLES]...`
```
Comment (zenatrick) [inline docs/UserGuide.adoc:2270]: Revert

File: docs/UserGuide.adoc [hunk 3]
```diff
@@ -595,11 +595,11 @@ Outcome:
 ****
 
 ==== Find activities by name: `findactivity`
-You can use this command to find activities by the name.
+You can use this command to find activities by title.
```
Comment (zenatrick) [inline docs/UserGuide.adoc:598]: Revert

File: docs/UserGuide.adoc [hunk 4]
```diff
@@ -595,11 +595,11 @@ Outcome:
...
 
 *Format:*
 
-`findactivity NAME [MORE_NAMES]...`
+`findactivity TITLE [MORE_TITLE]...`
```
Comment (zenatrick) [inline docs/UserGuide.adoc:602]: Revert

File: docs/UserGuide.adoc [hunk 5]
```diff
@@ -2357,19 +2363,19 @@ This feature will allow you to import and export your trip data in different for
...
 
-|*Find Packing List Items by Category*: `finditemcategory KEYWORD [MORE_KEYWORDS]...`
+|*Find Packing List Items by Category*: `finditemcategory CATEGORY [MORE_CATEGORIES]...`
 |`finditemcategory swimming clothes`
 
 |*Clear Packing List*: `clearitem`
 |`clearitem`
 
-|*Add Preset*: `addpreset CATEGORY`
+|*Add Preset*: `addpreset PRESET_NAME`
```
Comment (zenatrick) [inline docs/UserGuide.adoc:2378]: I believe the top and in the app is written is `CATEGORY` and I asked Cat yesterday which one to use

**Pull request merged**